joern-cli/frontends/javasrc2cpg/src/test/scala/io/joern/javasrc2cpg/querying/CallTests.scala

```diff
@@ -12,6 +12,22 @@ import overflowdb.traversal.toNodeTraversal
 
 class NewCallTests extends JavaSrcCode2CpgFixture {
 
+  "calls with unresolved receivers should have the correct fullnames" in {
+    val cpg = code("""
+        |import a.*;
+        |
+        |class Test {
+        |
+        |  void test() {
+        |    foo().bar();
+        |  }
+        |}
+        |""".stripMargin)
+
+    cpg.call.name("foo").typeFullName.l shouldBe List("ANY")
+    cpg.call.name("foo").methodFullName.l shouldBe List("Test.foo:<unresolvedSignature>(0)")
```

Comment:
In the fullName here, we probably shouldn't assume `foo` is a method in `Test`, but this is working as "intended" and should be fixed separately